area4/__init__.py

```diff
@@ -65,7 +65,33 @@ def divider(self, number) -> str:
                 return self.dividers[number].split("\n")[0]
             except IndexError:
                 raise ValueError('That divider doesn\'t exist!')
+    
+    
+    def logDivider(self, div, *stuff) -> str:
+        """
```

Comment:
@N8python can you review the documentation I just put in and see if it is accurate?

Reply:
It's accurate.